src/app/app-routing.module.ts

```diff
@@ -56,6 +58,13 @@ const routes: Routes = [
     component: ChatComponent,
     canActivate: [AuthGuard],
   },
+
+  {
+    path: 'chat-detail/:id',
+    component: ChatDetailComponent,
+    canActivate: [AuthGuard],
+  },
+
```

Comment:
```suggestion
```
Clean code that follows predictable patterns is extremely important in larger projects.